assets/jsx/hasher.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import filesize from 'filesize'
+
+
+class Hasher extends React.Component {
```

Comment:
The name `Hasher` to me sounds like it actually does the work. Can we rename it to be something like `HashStatus`?